examples/python/face_tracking/README.md

```diff
@@ -15,44 +16,179 @@ thumbnail_dimensions = [480, 480]
+```python
+rr.log(
+    "video/image",
+    rr.Image(frame).compress(jpeg_quality=75)
+)
+```
+
+## Face Landmark Points
+Logging the face landmarks involves specifying connections between the points, extracting face landmark points and logging them to the Rerun SDK.
+The 2D points are visualized over the video/image for a better understanding and visualization of the face.
+The 3D points allows the creation of a 3D model of the face reconstruction for a more comprehensive representation of the face.
+
+The 2D and 3D points are logged through a combination of two archetypes. First, a timeless
+[`ClassDescription`](https://www.rerun.io/docs/reference/types/datatypes/class_description) is logged, that contains the information which maps keypoint ids to labels and how to connect
+the keypoints. Defining these connections automatically renders lines between them.
+Second, the actual keypoint positions are logged in 2D and 3D as [`Points2D`](https://www.rerun.io/docs/reference/types/archetypes/points2d) and [`Points3D`](https://www.rerun.io/docs/reference/types/archetypes/points3d) archetypes, respectively.
+
+### Label Mapping and Keypoint Connections
+
+An annotation context is logged with one class ID assigned per facial feature. The class description includes the connections between corresponding keypoints extracted from the MediaPipe face mesh solution.
+A class ID array is generated to match the class IDs in the annotation context with keypoint indices (to be utilized as the class_ids argument to rr.log).
+```python
+# Initialize a list of facial feature classes from MediaPipe face mesh solution
+classes = [
+    mp.solutions.face_mesh.FACEMESH_LIPS,
+    mp.solutions.face_mesh.FACEMESH_LEFT_EYE,
+    mp.solutions.face_mesh.FACEMESH_LEFT_IRIS,
+    mp.solutions.face_mesh.FACEMESH_LEFT_EYEBROW,
+    mp.solutions.face_mesh.FACEMESH_RIGHT_EYE,
+    mp.solutions.face_mesh.FACEMESH_RIGHT_EYEBROW,
+    mp.solutions.face_mesh.FACEMESH_RIGHT_IRIS,
+    mp.solutions.face_mesh.FACEMESH_FACE_OVAL,
+    mp.solutions.face_mesh.FACEMESH_NOSE,
+]
+
+# Initialize class descriptions and class IDs array
+self._class_ids = [0] * mp.solutions.face_mesh.FACEMESH_NUM_LANDMARKS_WITH_IRISES
+class_descriptions = []
+
+# Loop through each facial feature class
+for i, klass in enumerate(classes):
+    # MediaPipe only provides connections for class, not actual class per keypoint. So we have to extract the
+    # classes from the connections.
+    ids = set()
+    for connection in klass:
+        ids.add(connection[0])
+        ids.add(connection[1])
+
+    for id_ in ids:
+        self._class_ids[id_] = i
+
+    # Append class description with class ID and keypoint connections
+    class_descriptions.append(
+        rr.ClassDescription(
+            info=rr.AnnotationInfo(id=i),
+            keypoint_connections=klass,
+        )
+    )
```

Comment:
`class`, also remove `klass` from dictionary again?

Reply:
confirmed, `class` doesn't work

Reply:
Oh, makes sense. I'd probably use `class_` instead, but it doesn't matter.